site/src/components/Combobox/Combobox.stories.tsx

```diff
@@ -103,7 +103,8 @@ export const SearchAndFilter: Story = {
 				screen.queryByRole("option", { name: "Kotlin" }),
 			).not.toBeInTheDocument();
 		});
-		await userEvent.click(screen.getByRole("option", { name: "Rust" }));
+		// Accessible name includes both image alt text and text content: "Rust Rust"
+		await userEvent.click(screen.getByRole("option", { name: "Rust Rust" }));
```

Comment:
Since we changed the icon image to be an ExternalImage with `alt={option.displayName}` https://github.com/coder/coder/pull/19063/files#diff-f7c0d4d4c43b6dd17458ff44a1a0dfdccfae367b09ffe41fdbe09c48c151e466R126
The name includes both the image alt text (`Rust`) + the text content (`Rust`). Let me know if this is ok, or if there is a better way to filter out this element.

Reply:
@Parkreiner Thank you for letting me know about this, this is really helpful context 👍 

You're right: this PR introduced `alt={option.displayName}`, which led to the repeated accessible name (e.g. "Rust Rust"). I hadn’t realized that would have a direct impact on screen readers, that's super useful to know.

I’ll go ahead and:
* Revert the `alt` attribute to use an empty string.
* Update the tests to reflect the correct accessible names without duplication.

Appreciate you taking the time to call this out 🙂